tests/e2e/tests/02-enable-serve-via-traffic/send_traffic.sh

```
@@ -1,12 +1,181 @@
+if [ "$POD_STATUS" != "Running" ]; then
+    echo "${YELLOW}WARNING: CURL Pod is not in Running state. Describing pod...${NC}"
+    kubectl describe pod "$POD_NAME" -n "$NAMESPACE" || true
+fi
+
+echo ""
+echo "${CYAN}--- Starting Traffic Test ---${NC}"
+
+failure_count=0
+
+for i in $(seq 1 $MAX_RETRIES); do
+    echo "\n${CYAN}--- Request $i/$MAX_RETRIES ---${NC}"
```

Shellcheck (0.10.0)

[info] echo may not expand escape sequences. Use printf.

(SC2028)